package.json

```diff
@@ -112,7 +112,7 @@
     "start:test": "run-p --race svelte:watch proxy:start:test electron:start",
     "test": "TZ='UTC' yarn test:unit && TZ='UTC' yarn test:integration",
     "test:integration": "TZ='UTC' run-p --race proxy:start:test wait:test",
-    "test:integration:debug": "RUST_LOG=proxy=debug TZ='UTC' run-p --race svelte:watch proxy:start:test wait:debug",
+    "test:integration:debug": "TZ='UTC' run-p --race svelte:watch proxy:start:test wait:debug",
```

Comment:
Why do we want to remove this?

Reply:
It's incorrect syntax and hid relevant log lines from proxy, i.e. log level `error`.

Reply:
Is there a way to enable all log levels for `test:integration:debug`?

Reply:
We could do `RUST_LOG=debug` but that trickles down to all the dependencies and will be extremely noisy. One sane default could be to enable it only for our local crates with `RUST_LOG=api=debug,coco=debug`.